api/src/pcapi/connectors/serialization/acceslibre_serializers.py

```diff
@@ -57,6 +57,8 @@ class Config:
     @classmethod
     def from_orm(cls, accessibility_infos: AccessibilityInfo) -> "ExternalAccessibilityDataModel":
         accessibility_data = super().from_orm(accessibility_infos)
+        if not accessibility_infos:
+            return ExternalAccessibilityDataModel()
```

Comment:
Puisqu'on retourne prématurément sans utiliser `accessibility_data`, est-ce que ça ne devrait pas fait en tout début de fonction ? 

```suggestion
        if not accessibility_infos:
            return ExternalAccessibilityDataModel()
        accessibility_data = super().from_orm(accessibility_infos)
```